decidim-core/app/models/decidim/pad.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
```

Comment:
I think this is not a model in the Rails context as it's not saving any data to the database. Could we move this e.g. to `decidim-core/app/services`? Or maybe even to `decidim-core/lib/decidim/core/etherpad`?

Also, let's rename it to `Etherpad::Pad` to better describe the service it's meant for because this is pretty bound to the `Etherpad` service with the API call.